Jenkinsfile

```diff
@@ -0,0 +1,21 @@
+pipeline {
+    agent label:"docker", docker: "maven:3.3.9-jdk-7"
+
+    environment {
+      MAVEN_OPTS = "-XX:MaxPermSize=256m -Xmx1024m"
```

Comment:
Of course the `MaxPermSize` would be unnecessary on 8. Whether `Mx` is really needed here is TBD. Offhand I would guess that the JRE’s ergonomics are blind to the fact that you are running in a container, and thus the default max heap is going to vary according to host environment.